source/javascripts/application.js

```diff
@@ -0,0 +1 @@
+//= require vendor/modernizr
```

Comment:
I started wondering whether we could do away with Modernizr.js by using @supports (display: flex) instead to override any non-flex styles but I guess IE11 for one would be an issue as it supports flex but doesn’t support CSS.supports() API.

Should we document somewhere what things (like this pane component) require Modernizr so that the user can choose not to include it unnecessarily?